src/psbt.cpp

```diff
@@ -38,8 +42,23 @@ bool PartiallySignedTransaction::Merge(const PartiallySignedTransaction& psbt)
 
 bool PartiallySignedTransaction::IsSane() const
 {
-    for (PSBTInput input : inputs) {
+    for (unsigned int i = 0; i < inputs.size(); ++i) {
+        const PSBTInput& input = inputs[i];
         if (!input.IsSane()) return false;
+        // Check issuance is empty in CTxOut but set in input
+        if (!tx->vin[i].assetIssuance.IsNull()) {
+            if (!tx->vin[i].assetIssuance.nAmount.IsNull()) return false;
+            if (!input.issuance_value && input.issuance_value_commitment.IsNull()) return false;
+        }
+        // Check whether any input is blinded
```

Comment:
The comment is at the end of the loop. Usually the comment describes the code that is below (or left of) the comment. Does this comment refer to the code that is supposed to be here, but removed ?

Reply:
Removed